CMakeLists.txt

```diff
@@ -12,7 +12,6 @@ option(MATLAB_BINDINGS "Compile MATLAB bindings if MATLAB is found." OFF)
 option(TEST_VERBOSE "Run test cases with verbose output." OFF)
 option(BUILD_TESTS "Build tests." ON)
 option(BUILD_CLI_EXECUTABLES "Build command-line executables." ON)
-option(BUILD_PYTHON_BINDINGS "Build Python bindings." ON)
```

Comment:
I liked the structure; every option is easily parsable at the first glance since there is no extra if condition. I think we could per default use: ``option(BUILD_PYTHON_BINDINGS "Build Python bindings." ON)`` and reset the option if we build under windows. not sure this would result in an redefinition error.

Reply:
I‘m trying but my cmd keeps saying that `CMake Error: The source directory "D:/projects/mlpack-master/build" does not appear to contain CMakeLists.txt.`
How could that happen? I do have CMakeList

Reply:
Ah, when you run CMake, make sure the source directory is set correctly.  Like if you are running from the command line in the build directory, use `cmake ../` not `cmake .`, for instance.  Let me know if I can clarify.